test/mpi/comm/cmsplit_type.c

```diff
@@ -22,7 +22,7 @@ static const char *split_topo[] = {
 };
 
 static const char *split_topo_network[] = {
-    "switch_level:2", "subcomm_min_size:2", NULL
+    "switch_level:2", "subcomm_min_size:2", "min_mem_size:512", NULL
```

Comment:
What's the unit here?

Reply:
The unit of memory is in bytes. I'll add comments and description in the PR stating this.